python/variable.cpp

```diff
@@ -165,14 +177,6 @@ void init_variable(py::module &m) {
       .def("__deepcopy__",
            [](Variable &self, py::dict) { return Variable(self); })
       .def_property_readonly("dtype", &Variable::dtype)
-      .def(py::self + double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self - double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self * double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self / double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self += double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self -= double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self *= double(), py::call_guard<py::gil_scoped_release>())
-      .def(py::self /= double(), py::call_guard<py::gil_scoped_release>())
```

Comment:
Were these never needed or is it related to the addition of the `pyobject`

Reply:
It is unrelated. They were initially needed, but were made superfluous by a recent addition (below) which adds bindings for different scalar types. These are then just "overridden".